clients/src/main/java/org/apache/kafka/common/requests/ApiVersionsResponse.java

```diff
@@ -271,4 +273,48 @@ public static ApiVersion toApiVersion(ApiKeys apiKey) {
             .setMinVersion(apiKey.oldestVersion())
             .setMaxVersion(apiKey.latestVersion());
     }
+
+    public static ApiVersionsResponse apiVersionsResponse(
+        Integer throttleTimeMs,
+        RecordVersion minRecordVersion,
+        Features<SupportedVersionRange> latestSupportedFeatures,
+        NodeApiVersions controllerApiVersions,
+        ListenerType listenerType
+    ) {
+        return apiVersionsResponse(
+            throttleTimeMs,
+            minRecordVersion,
+            latestSupportedFeatures,
+            Features.emptyFinalizedFeatures(),
+            ApiVersionsResponse.UNKNOWN_FINALIZED_FEATURES_EPOCH,
+            controllerApiVersions,
+            listenerType
+        );
+    }
+
+    public static ApiVersionsResponse apiVersionsResponse(
+        Integer throttleTimeMs,
```

Comment:
Please use int and long rather than `Integer` and `Long`, unless these are nullable